src/app.module.ts

```diff
@@ -2,14 +2,23 @@ import { Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { ConfigModule, ConfigService } from '@nestjs/config';
 
-import { AuthModule } from './auth/auth.module';
-import { DoctorsModule } from './doctors/doctors.module';
-import { UserEntity } from './auth/user.entity';
-import { DoctorEntity } from './doctors/doctor.entity';
+import { AuthModule } from 'auth/auth.module';
+import { UserEntity } from 'auth/user.entity';
+
+import { DoctorsModule } from 'doctors/doctors.module';
+import { DoctorEntity } from 'doctors/doctor.entity';
```

Comment:
why absolute imports?

Reply:
For using dependencies inside of module I use relative path for keep modularity.
For access to outside world better to use absolute paths, its mush simple for refactoring or transferring

@macc4 Agree, for app.module.ts it doesn't have a lot of sense